docs/operator-manual/applicationset/Controlling-Resource-Modification.md

```diff
@@ -14,17 +14,50 @@ To enable dry-run, add `--dryrun true` to the ApplicationSet Deployment's contai
 
 See 'How to modify ApplicationSet container parameters' below for detailed steps on how to add this parameter to the controller.
 
-### Policy - `create-only`: Prevent ApplicationSet controller from modifying or deleting Applications
+### Managed Applications modification Policies
 
 The ApplicationSet controller supports a parameter `--policy`, which is specified on launch (within the controller Deployment container), and which restricts what types of modifications will be made to managed Argo CD `Application` resources.
 
-The `--policy` parameter takes one of the following valid values: `sync`, `create-only`, `create-update`, and `create-delete`. (`sync` is the default, which is used if the `--policy` parameter is not specified; the other policies are described below).
+The `--policy` parameter takes four values: `sync`, `create-only`, `create-delete`, and `create-update`. (`sync` is the default, which is used if the `--policy` parameter is not specified; the other policies are described below).
+
+It is also possible to set this policy per ApplicationSet.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+spec:
+  # (...)
+  syncPolicy:
+    applicationsSync: create-only # create-update, create-delete sync
+
+```
+
+- Policy `create-only`: Prevents ApplicationSet controller from modifying or deleting Applications.
+- Policy `create-update`: Prevents ApplicationSet controller from deleting Applications. Update is allowed.
+- Policy `create-delete`: Prevents ApplicationSet controller from modifying Applications. Delete is allowed.
```

Comment:
I think I missed this, should we add `sync` here as an option as well?

Reply:
I added the policy sync.